docker/demisto-sdk/Dockerfile

```diff
@@ -24,29 +24,15 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /
     && apt-get update && apt-get install docker-ce-cli -y
```

Comment:
This package docker-ce-cli should be included in the previous command (this also reduce the number of layers)

Reply:
but we need first the prev commands for it to work?